Routes/admin.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const { body } = require('express-validator');
+
+const router = express.Router();
+
+const {
+  getAddProduct,
+  postAddProduct,
+  getAdminProducts,
+  getUpdateMenu,
+  updateProduct,
+  deleteProduct
+} = require('../controllers/admin');
+const isAuth = require('../middleware/is-auth');
+
+router.post(
+  '/product',
+  isAuth,
+  [
+    body('productName').isString().trim().withMessage('Enter correct Title'),
+    body('price').isFloat().withMessage('Price should be a floating number.'),
+    body('description')
+      .isLength({ min: 3, max: 400 })
+      .trim()
+      .withMessage('Description should contain atlease 3 characters.')
+  ],
+  postAddProduct
+);
+
+router.get('/products', isAuth, getAdminProducts);
+
+router.get('/add-product', isAuth, getAddProduct);
+
+router.post('/products/edit', isAuth, getUpdateMenu);
```

Comment:
Improve conventions here to define your route. no need to add `/edit` at the end.